test/studies/champs/sub_test

```diff
@@ -4,15 +4,34 @@
 
 CHAMPS_BRANCH=${CHAMPS_BRANCH:-development}
 
+CWD=$(cd $(dirname ${BASH_SOURCE[0]}) ; pwd)
+source $CWD/functions.bash
+
+if [ ! -z "$CHAMPS_QUICKSTART" ]; then
+  if [ -z "$CHPL_HOME" ]; then
+    echo "[Error: \$CHPL_HOME is unset]"
```

Comment:
Do you want to have your script return at this point given that it's errored out?

Reply:
https://github.com/chapel-lang/chapel/pull/23444/commits/3128ea4f36140fbbf9322078c50e9caa88f31848